meta-resin-common/recipes-core/util-linux/util-linux_%.bbappend

```diff
@@ -1,2 +1,11 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+python() {
+	packageVersion = d.getVar('PV', True)
+	srcURI = d.getVar('SRC_URI', True)
+	if packageVersion >= '2.28':
```

Comment:
May want to do it like this: https://github.com/resin-os/meta-resin/pull/749/commits/d9f2998fb132324753dd36b57406dce6916ef676